packages/runtime/src/runtimes/react/controls/rich-text-v2/EditableTextV2/editable-text-v2.tsx

```diff
@@ -52,8 +61,16 @@ type Props = {
 export function EditableTextV2({ text, definition, control }: Props) {
   const plugins = useMemo(() => {
     const plugins = [
-      ...(definition?.config?.plugins ?? []),
-      ...(definition?.config?.mode === RichTextV2Mode.Inline ? [InlineModePlugin] : []),
+      /**
+       * TODO: we are manually referencing our default plugins for each mode here because
+       * Referencing the real LinkPlugin causes a circular dependency.
+       * When circular dependencies calm down we should update the plugin definition to use real plugins,
+       * and just use the plugins that are defined by our config.
+       */
+      // ...(definition?.config?.plugins ?? []),
```

Comment:
What's the problem with the circular dependencies? Are you trying to avoid them or are they resulting in a build time or runtime error?

Reply:
I am trying to avoid them, and they are resulting in a build time error.

Referencing the slate plugins in `/slate` from my control creates an error because the `Link` plugin references `useElementId`.

Reply:
Ok. Can you remove the circular dependencies?

Reply:
Maybe :) 

I spent some time on it yesterday, and I don't know what removing them entails. The thing that is triggering the circular dependency is referencing `runtimes/react/index.ts` which is importing a lot of things.